srcpkgs/dqlite/template

```diff
@@ -1,24 +1,24 @@
 # Template file for 'dqlite'
 pkgname=dqlite
-version=0.2.6
+version=1.1.0
 revision=1
 build_style=gnu-configure
 hostmakedepends="pkg-config autoconf automake libtool"
-makedepends="libuv-devel sqlite-replication-devel"
+makedepends="sqlite-replication-devel raft-devel libco-devel"
 depends="sqlite-replication"
 short_desc="Distributed SQLite database using the Raft algorithm"
 maintainer="Cameron Nemo <camerontnorman@gmail.com>"
 license="Apache-2.0"
```

Comment:
LGPLv3 https://github.com/canonical/dqlite/commit/fe0fbed8ca717947f90bf2018b28d06644d307b0

Reply:
Could not find an SPDX for LGPL-3.0-only with linking exception, so used custom. Also xlint will still complain due to https://github.com/leahneukirchen/xtools/pull/146 .